CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## unreleased
 
+- Relax comment parsing; control characters are again permitted.
```

Comment:
Change to: ...; most control characters are again permitted.

Reply:
Will do. You'll see it in a bit.